features/0434-outofband/README.md

```diff
@@ -0,0 +1,417 @@
+
+### Standard Out of Band Message Encoding
+
+Using a standard out of band message encoding allows for easier interoperability between multiple projects and software platforms. Using a URL for that standard encoding provides a built in fallback flow for users who are unable to automatically process the message. Those new users will load the URL in a browser as a default behavior, and may be presented with instructions on how to install software capable of processing the message. Already onboarded users will be able to process the message without loading in a browser via mobile app URL capture, or via capability detection after being loaded in a browser.
+
+The standard out of band message format is a URL with a Base64URLEncoded json object as a query parameter.
+
+The URL format is as follows, with some elements described below:
+
+```text
+https://<domain>/<path>?c_i=<outofbandMessage>
+```
+
+`<domain>` and `<path>` should be kept as short as possible, and the full URL should return human readable instructions when loaded in a browser. This is intended to aid new users. The `c_i` query parameter is required and is reserved to contain the invitation string. Additional path elements or query parameters are allowed, and can be leveraged to provide coupons or other promise of payment for new users.
+
+> To do: We need to rationalize this approach `https://` approach with the use of a special protocol (e.g. `didcomm://`) that will enable handling of the URL on mobile devices to automatically invoke an installed app on both Android and iOS. A user must be able to process the out of band message on the device of the agent (e.g. when the mobile device can't scan the QR code because it is on a web page on device).
+
+The `<outofbandMessage>` is an agent plaintext message (not a DIDComm message) that has been base64 url encoded.
+
+```javascript
+outofband_message = b64urlencode(<outofbandMessage>)
+```
+
+During encoding, whitespace from the json string should be eliminated to keep the resulting invitation string as short as possible.
+
+#### Example Out of Band Message Encoding
+
+Invitation:
+
+```json
+{
+  "@type": "https://didcomm.org/outofband/1.0/invitation",
+  "@id": "69212a3a-d068-4f9d-a2dd-4741bca89af3",
+  "label": "Alice",
+  "protocols": ["https://didcomm.org/did-exchange/1.0", https://didcomm.org/connections/1.0]
```

Comment:
```suggestion
  "protocols": ["https://didcomm.org/did-exchange/1.0", "https://didcomm.org/connections/1.0"],
```

Reply:
Fixed all the way through.  As well "did-exchange" should have been "didexchange", so that was changed as well.  Regenerated the rest of the examples based on the new message.